backend/i18n/frontend_it.json

```diff
@@ -1,6 +1,6 @@
 {
-  "api.contentApi": "Content API",
```

Comment:
Please check the docs

Reply:
ah sorry, I have to edit the source jsons